defs/gmake.mk

```diff
@@ -472,6 +472,8 @@ rubyspec-capiext: $(patsubst %.c,$(RUBYSPEC_CAPIEXT)/%.$(DLEXT),$(notdir $(wildc
 
 ifeq ($(ENABLE_SHARED),yes)
 exts: rubyspec-capiext
+else ($(filter darwin%,$(target_os)),)
```

Comment:
```suggestion
else if ($(filter darwin%,$(target_os)),)
```

Reply:
Ah, thanks for catching this. It should be `ifeq`. I've pushed the correct syntax.